### DISCUSSION TITLE
[PSD1] Exercise 1.21

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#123] [Esercizi (A-L)]
**Exercise 1.21** Convert the following two’s complement binary numbers to decimal
(a) $1010_2$
(b) $110110_2$
(c) $01110000_2$
(d) $10011111_2$

**Comment by niccolo-fato** [comment]
![photo_5870846455839833123_y](https://user-images.githubusercontent.com/73995334/210087419-af958f82-a2b3-48f6-a8da-83713eac3692.jpg)
